local-registry/Cargo.toml

```diff
@@ -153,14 +124,9 @@ num-integer = "0.1.45"
 numtoa = "0.2.4"
 num-traits = "0.2.15"
 once_cell = "1.18.0"
-onig = "6.4.0"
 ordered-float = "3.7.0"
-out = "8.0.0"
 pathfinding = "4.3.0"
-pcre = "0.2.3"
```

Comment:
You sure this isn't used?

Reply:
I would be very surprised. This crate had its last release about 7 years ago, that's not long after Rust 1.0 was released. The `regex` crate has been the standard for many years (and is maintained by the official organization).

There seems to be a maintained version [pcre2](https://docs.rs/pcre2/latest/pcre2/), but I don't see a reason to have more than one regex library.

Reply:
I considered scraping the community solutions for crates used, but decided not to abuse your servers too much :joy:

Reply:
Okay great!